### PR TITLE
Issue#83 doc, adds test contract to LIC9 test-cases

### LINKED ISSUE #83: Add contract for `LIC9` tests

### PR DESCRIPTION
**Pull request by Jonassavas**
Fixes #83 adds test contract to all LIC9 test-cases with pre- and postconditions.

### CODE REVIEW

File: src/test/java/com/decide/DecideTest.java [hunk 1]
```diff
@@ -383,33 +383,50 @@ public void LIC8TestNotInCircle(){
 
 
     @Test
-    /**Tests false boundary values for LIC8 where the number of points
-     * can not be less than 5 and a_pts and b_pts can not be lower than 1.
-    */
+    /**
+     * Requirements: See `LIC9` documentation
+     * Contract:
+     *      Precondition: numPoints can not be less than 5: numPoints = 4,
```
Comment (FalkWasTaken) [inline src/test/java/com/decide/DecideTest.java:389]: The preconditions should probably have the format `x is less than y` instead of just describing the boundary.

**Review by FalkWasTaken — CHANGES_REQUESTED** [review]
See my comment regarding the boundary test.

File: src/test/java/com/decide/DecideTest.java (hunk 1 above)
Comment (Jonassavas) [reply src/test/java/com/decide/DecideTest.java]: Yes, should be fixed now!

**Review by FalkWasTaken — APPROVED** [review]
Super!
